packages/components/src/components/hds/table/th-button-sort.ts

```diff
@@ -5,10 +5,29 @@
 
 import Component from '@glimmer/component';
 import { guidFor } from '@ember/object/internals';
+import {
+  HdsTableThSortOrder as HdsTableThSortOrderValues,
+  HdsTableThSortOrderIcons as HdsTableThSortOrderIconValues,
+  HdsTableThSortOrderLabels as HdsTableThSortOrderLabelValues,
+} from './types.ts';
```

Comment:
we don't need aliases here, right? we can update the types to have the `Values` suffix

Reply:
will do ... TypeScript can infer context so really don't need suffix at all.